package.json

```diff
@@ -121,7 +121,10 @@
                 },
                 "args": [
                     "/debugAdapter",
-                    "/logLevel:Verbose"
+                    "/logLevel:Verbose",
+                    "/hostName:\"Visual Studio Code Host\"",
+                    "/hostProfileId:Microsoft.VSCode",
+                    "/hostVersion:0.5.0"
```

Comment:
The debugger configuration is a little unfortunate since we have to update the version in 3 places in this file now.  We'll need a version bump script in the near future.